packages/@aws-cdk/cfnspec/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.0.0",
   "scripts": {
     "update": "cdk-build && /bin/bash build-tools/update.sh",
+    "update-metrics": "/bin/bash build-tools/update-metrics.sh",
```

Comment:
Should we just update as part of our normal cfn-spec import (save us yet another ops task)

Reply:
Can't. There is no automated source (yet) to get this file from.